scripts/android-studio-build.gradle

```diff
@@ -0,0 +1,27 @@
+task buildAndroidStudioRelease(type: Zip) {
+    archiveName 'androidStudioRelease.zip'
```

Comment:
well really this is a zip for us to transfer it to their codebase, versions are still in the jars BUT, yes I can do that.